src/java.security.jgss/share/native/libj2gss/GSSLibStub.c

```diff
@@ -198,7 +198,7 @@ gss_channel_bindings_t newGSSCB(JNIEnv *env, jobject jcb) {
 
   cb = malloc(sizeof(struct gss_channel_bindings_struct));
   if (cb == NULL) {
-    throwOutOfMemoryError(env,NULL);
+    gssThrowOutOfMemoryError(env,NULL);
```

Comment:
While you're fixing this, add a space between arguments, e.g. `(env,NULL) `becomes `(env, NULL)`.

Reply:
Done, thanks.